css/main.css

```diff
@@ -380,4 +382,45 @@ input {
 	height: 240px;
 	margin: 0 auto;
 	margin-top: 2%;
+}
+
+/* Social Media Call to Action */
+
+.social-bkg {
+	background-color: rgba(122, 242, 132, 0.461);
+	border-radius: 2rem;
+	width: 90%;
+	margin: auto;
+	padding: 1em;
+}
+
+.socialmedia {
+	text-align: center;
+}
+
+.socialmedia h3, h4 {
+	font-weight: 800;
+	text-align: center;
+}
+
+.social-pic img {
+	width: 30%;
+	border-radius: 2rem;
+	padding: 1rem;
+}
+
+.social-links, .social-pic {
+	display: inline-block;
+}
+
+.social-links a {
+	/* margin: auto; */
```

Comment:
Line 417 can be deleted if not needed.

Reply:
This is an issue that needs to be resolved. Some user are seeing the caption falling below the carousel indicators and others are not. I know Nick Gagan and I all see the captions correctly, but you and I believe Margaret are seeing them cut off.
@billythakidd04 how should we proceed with this issue? I did some Google searches on the issue, but saw nothing specifically for Bootstrap5. I can adjust the CSS ,  but I don't think that will make the position consistent to all of the users. Below is a screen capture of what I see in the same slide.
![image](https://user-images.githubusercontent.com/78501210/113041909-86418800-9168-11eb-97a0-72e700e1c461.png)

Reply:
I don't know what you see when it is adjusted to a phone screen, but the images scale strange, and the text disappears. I would go into CSS at this point, but as it is not ideal I also wonder if there is a bootstrap class that could fix it. 

![image](https://user-images.githubusercontent.com/78566744/113051608-2c46bf80-9174-11eb-892b-b478b32bc1ad.png)

Reply:
> I don't know what you see when it is adjusted to a phone screen, but the images scale strange, and the text disappears. I would go into CSS at this point, but as it is not ideal I also wonder if there is a bootstrap class that could fix it.
> 
The text disappearing is a Responsive Text issue that I'm seeing on all of the pages except for the Menu page. This will need to be adjusted throughout the site.
The spacing between the Hero Header and Carousel is likely due to padding being set in pixels rather than percent or rem. This will also need to be addressed throughout the site.
Both issues are outside of the scope of this issue. New issues will need to be opened to address them.

We still need to address the issue of the Carousel text falling below the indicators for some users.

Reply:
I would see if there's a bootstrap class that allows you to set the text to the center of the container. That should solve the issue and look more correct. check out the examples page for bootstrap. I think theres something similar on there.

As for the px vs % issues, @vricci518 is absolutely correct and those tickets should be created immediately.